### ISSUE TITLE
使用visa数据集训练模型，测试效果不理想，与官方已训练模型测试效果相差甚远

### ISSUE THREAD

**Issue by lpl071528** [zqhang/AnomalyCLIP#7]
Hello author, I tested the Visa dataset using a model that has been trained in engineering and the results were good. However, when I used the Visa dataset, created a meta.json file, and trained for 15 epochs, the testing results were very poor. What is the reason for this?
![1](https://github.com/zqhang/AnomalyCLIP/assets/85934200/c6f7fb5c-2566-4b41-8a62-8367a4a20a2a)
![2](https://github.com/zqhang/AnomalyCLIP/assets/85934200/1c73bb82-9f2d-481e-8c9c-05e65d3bf529)

**Comment by TIHOHIT** [comment]
Maybe the epochs are too small?

**Comment by Wangjinhong1998** [comment]
Similarly, when I train the replicated model myself and test it on the VISA dataset, the results are not as good as those of the author, with a difference of about 1 to 2 percentage points

**Comment by Keep-git** [comment]
我是用visa做训练集，在mvtec上进行测试，与作者的pixel_auroc上的精度相差了4%很多，请问有发现是怎么回事吗？

**Comment by anonymous-kat** [comment]
> Maybe the epochs are too small?

I dont think it is the problem of epochs. I remember in my training, the loss always converges pretty fast.

**Comment by anonymous-kat** [comment]
> 我是用visa做训练集，在mvtec上进行测试，与作者的pixel_auroc上的精度相差了4%很多，请问有发现是怎么回事吗？

我是在mvtec训练，visa测试
PRO会错0.06-0.07，其他几个指标错0.02左右
